src/routes/chatRoomRoutes.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const chatRoomsQueries = require('../db/queries/chatRoomsQueries');
+
+const routes = () => {
+  const chatRoomRoutes = express.Router();
```

Comment:
ok wow there let's stop here. you need to make 1 router file that has `const router = express.Router()` you are making your file structure way too big here, 1 router file 1 controllers folder